model.py

```diff
@@ -3250,6 +3250,8 @@ def set_summary(self, resource):
         # TODO: clean up the content
         if resource:
```

Comment:
I can put this in, but it won't let you remove a summary entirely if it's from another source - Identifier.evaluate_summary_quality will throw out the empty one. That's probably ok for now. Once composite editions are ready we'll have a better way to fix it.